@kiva/kv-components/vue/KvExpandableQuestion.vue

```diff
@@ -0,0 +1,124 @@
+<template>
+	<div>
+		<button
+			class="tw-w-full tw-py-2 tw-flex tw-justify-between"
+			@click="toggleFAQ"
+		>
+			<h3 class="tw-text-subhead tw-text-left">
+				{{ title }}
+			</h3>
+			<kv-material-icon
+				class="tw-w-4 tw-h-4"
+				:icon="open ? mdiChevronUp : mdiChevronDown"
+				@click="open = !open"
+			/>
+		</button>
+		<kv-expandable easing="ease-in-out">
+			<div
+				v-show="open"
+				class="tw-prose tw-pb-4 tw-pt-2"
+				v-html="content"
+			>
+			</div>
+		</kv-expandable>
+	</div>
+</template>
+
+<script>
+import {
+	computed,
+	onMounted,
+	watch,
+	ref,
+	toRefs,
+} from 'vue-demi';
+import {
+	mdiChevronDown,
+	mdiChevronUp,
+} from '@mdi/js';
+import { paramCase } from 'change-case';
+
+import KvExpandable from './KvExpandable.vue';
+import KvMaterialIcon from './KvMaterialIcon.vue';
+
+export default {
+	components: {
+		KvMaterialIcon,
+		KvExpandable,
+	},
+	props: {
+		/**
+		 * Question Title
+		* */
+		title: {
+			type: String,
+			default: '',
+		},
+		/**
+		 * Question Content - can accept raw html
+		* */
+		content: {
+			type: String,
+			default: '',
+		},
+		/**
+		 * Analytics Category
+		* */
+		analyticsCategory: {
```

Comment:
We can remove this prop. The analytics category for this should always be `faq`

Reply:
Nice!. @tiarascoleman note that is all lowercase `faq` which can be placed directly in the code.

Reply:
@eddieferrer any thoughts or changes we should make to the title in the analytics event?

Reply:
See my other comment